recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -195,10 +212,13 @@ def package_info(self):
             self.cpp_info.exelinkflags = ldflags
             self.cpp_info.sharedlinkflags = ldflags
 
+        # Breaking change since Conan >= 2.0.8
+        # Related to https://github.com/conan-io/conan/pull/14233
+        libdir_variable = "libdir1" if Version(conan_version) < "2.0" else "libdir1"
```

Comment:
Am i missing something here? `libdir1` is equals to `libdir1`

Reply:
My fault 😭 
```suggestion
        libdir_variable = "libdir1" if Version(conan_version) < "2.0" else "libdir"
```